modules/account-lib/src/coin/algo/transaction.ts

```diff
@@ -229,6 +229,10 @@ export class Transaction extends BaseTransaction {
         result.nonParticipation = this._algoTransaction.nonParticipation;
       }
     }
+    if (result.type === 'axfer' && result.to && result.amount) {
```

Comment:
```suggestion
    if (result.type === 'axfer' && result.to && result.amount && result.from) {
```

Reply:
I know this is a mandatory param, but an extra check could be great.

Reply:
I only include parameters that are optional on TxData. Parameter "from" is required on TxData so it must be there in order to build this interfase or will break before this condition